platform/linux-generic/arch/aarch64/odp/api/abi/cpu_inlines.h

```diff
@@ -28,12 +29,17 @@ extern _odp_cpu_cycles_global_t _odp_cpu_cycles_glob;
 
 static inline void _odp_cpu_pause(void)
 {
+#if defined(ENABLE_WFXT_CPU_PAUSE) && defined(__ARM_FEATURE_WFXT)
+	__asm__ volatile("sevl" : : : "memory");
```

Comment:
Why is the SEVL done here? Doesn't is basically defeat the purpose of the following WFET as the WFET would wake immediately and not wait for the timeout? If the SEVL is intentional and serves a purpose, then maybe that could be explained in a comment, unless it is just me who cannot see it. But if the SEVL stays, then why WFET instead of just WFE which would be available in much earlier arch levels?